sacred/utils.py

```diff
@@ -461,35 +474,50 @@ def print_filtered_stacktrace(filter_traceback=True):
 
 
 def format_filtered_stacktrace(filter_traceback=True):
+    """
+    Returns the traceback as `string`.
+
+    `filter_traceback` can be one of:
+        - 'always': always filter out sacred internals
+        - `True` or 'default': Default behaviour: filter out sacred internals
+                if the exception did not originate from within sacred, and
+                print just the internal stack trace otherwise
+        - `False` or 'never': don't filter, always print full traceback
+        - All other values will fall back to 'never'.
+    """
```

Comment:
The argument handling of `filtered_traceback` is a bit too lenient for my taste. I think I'd prefer allowing only `always`, `default` and `never` and raising an error on all other values. This breaks the backwards compatibility, but since this is only an internal method, it should be fine.